pkg/dockerfile/generator_test.go

```diff
@@ -10,9 +10,9 @@ import (
 	"github.com/replicate/cog/pkg/config"
 )
 
-func testInstallCog(generatedPaths []string) string {
-	return `COPY .cog/tmp/cog-0.0.1.dev-py3-none-any.whl /tmp/cog-0.0.1.dev-py3-none-any.whl
-RUN --mount=type=cache,target=/root/.cache/pip pip install /tmp/cog-0.0.1.dev-py3-none-any.whl`
+func testInstallCog(relativeTmpDir string) string {
```

Comment:
I'm not sure I like these tests that contain generated values. I almost got screwed by a false positive where `relativeTmpDir` was the empty string. We should be injecting this value into the generator.